packages/paste-website/src/component-examples/DisclosureExamples.ts

```diff
@@ -36,3 +36,36 @@ render(
   <StateHookDisclosure />
 )
 `.trim();
+
+export const StateHookToggleExample = `
+const StateHookToggleDisclosure = () => {
+  const [show, setShow] = React.useState(false);
+  const disclosure = useDisclosureState();
+  const toggleDisclosure = () => setShow(!show);
```

Comment:
NIT: Line 44 might be a good case to use functional updates.
e.g. `const toggleDisclosure = () => setShow(state => !state);` where the `state` argument refers to the most current value of `show`. This is useful to ensure that the state value `show` is not stale.
More information in the [React hooks docs](https://reactjs.org/docs/hooks-reference.html#functional-updates).